### PR TITLE
[spaceship][docs] update spaceship app store connect docs

### PR DESCRIPTION
**Pull request by rogerluan**
- [x] I've updated the documentation if necessary.

### Motivation and Context

This was exposed as an issue here: https://github.com/fastlane/fastlane/issues/17879#issuecomment-754101484

### Description

Simple docs updating. Removed a comment that said that something was not working (but it is), and update param name.

### CODE REVIEW

**Comment by rogerluan** [comment]
Actually I'm not sure whether the "not working" part should be removed. Looks like it doesn't work with ASC, but the same function does work with regular authentication credentials. 🤔

**Comment by joshdholtz** [comment]
@googlebot I consent.

**Review by joshdholtz — APPROVED** [review]
🔥

File: spaceship/docs/AppStoreConnect.md [hunk 1]
```diff
@@ -60,11 +60,11 @@ Spaceship::ConnectAPI::App.all.collect do |app|
 end
 
 # Create a new app
-// Not working yet
+# Currently only works with Apple ID login (not API Key)
```
Comment (rogerluan) [inline spaceship/docs/AppStoreConnect.md:63]: Cheers! 🎉 much more clear now!

**Pull request merged**